aws/request/handlers.go

```diff
@@ -20,22 +20,6 @@ type Handlers struct {
 	AfterRetry       HandlerList
 }
 
-// Copy returns of this handler's lists.
-func (h *Handlers) Copy() Handlers {
```

Comment:
Like @vburenin said, this happens a lot.

I believe this is a good case where the so called convenience of user trying to add a metric or a logging as you say should be taken away and have the user add the code for monitoring manually.

In @vburenin's case, this alone takes 3ms to happen for every single time. It's a third of the "single digit millisecond" "feature" of DynamoDB.